dolphinscheduler-common/pom.xml

```diff
@@ -313,6 +314,26 @@
 			</exclusions>
 		</dependency>
 
+		<dependency>
+			<groupId>org.apache.hadoop</groupId>
+			<artifactId>hadoop-aliyun</artifactId>
+			<version>${hadoop.aliyun.version}</version>
+			<exclusions>
+				<exclusion>
+					<groupId>org.apache.hadoop</groupId>
+					<artifactId>hadoop-common</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.codehaus.jackson</groupId>
+					<artifactId>jackson-core-asl</artifactId>
+				</exclusion>
+				<exclusion>
+					<groupId>org.codehaus.jackson</groupId>
+					<artifactId>jackson-mapper-asl</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
```

Comment:
adding a new jar requires e-mail instructions. You can refer to (https://dolphinscheduler.apache.org/zh-cn/docs/development/DS-License.html) and send an e-mail  to the community after completing the self-check.